inventree/part.py

```diff
@@ -256,3 +260,9 @@ def add_related(cls, api, part1, part2):
             logging.warning("Related failed")
             ret = False
         return ret
+
+    @classmethod
+    def delete_related(cls, api, part):
```

Comment:
Doesn't the existing "delete" function provide this functionality already? What exactly is the function here?

Reply:
Yes, I did not see this. Thanks  for the quality control. I am just starting. Sorry for the inconvenience.